server/views/_messages.njk

```diff
@@ -17,3 +17,12 @@
     }) }}
   {% endfor %}
 {% endif %}
+
+{% if successWithDetailMessages %}
+  {% for message in successWithDetailMessages %}
```

Comment:
I'd be halfway tempted to keep everything in the success array, and do something like `{% if success instanceof string %} ... {% else %} ... {% endif %}`, but this is definitely "have you considered..." rather than "this would be the right way to do it"

Reply:
I actually didn't consider this, I like this approach better.